app/main/views/briefs.py

```diff
@@ -0,0 +1,123 @@
+        "services/edit_submission_section.html",
+        framework=framework,
+        service_data={},
+        section=section,
+        **dict(main.config['BASE_TEMPLATE_DATA'])
+    ), 200
+
+
+# Add a create route
+@main.route('/opportunities/<int:brief_id>/responses/create', methods=['POST'])
+@login_required
+def create_new_brief_response(brief_id):
+    """Hits up the data API to create a new brief response."""
+
+    if not is_supplier_eligible_for_this_brief(data_api_client, current_user.supplier_id, brief_id):
+        abort(403)
+
+    if not has_supplier_already_submitted_a_brief_response(data_api_client, current_user.supplier_id, brief_id):
+        # TODO redirect to summary of brief response page with flash message
+        abort(404)
+
+    brief = get_live_brief(data_api_client, brief_id)
+
+    framework, lot = get_framework_and_lot(
+        data_api_client, brief['frameworkSlug'], brief['lotSlug'], allowed_statuses=['live'])
+
+    content = content_loader.get_manifest(framework['slug'], 'edit_brief_response').filter({'lot': lot['slug']})
+    section = content.get_section(content.get_next_editable_section_id())
+    response_data = section.get_data(request.form)
+
+    try:
+        data_api_client.create_brief_response(
+            brief_id, current_user.supplier_id, response_data, current_user.email_address
+        )['briefResponses']
+    except HTTPError as e:
+        # `errors` doesn't take into account error messages in (nested) boolean_list question fields
+        errors = section.get_error_messages(e.message, lot['slug'])
+        new_errors = OrderedDict()
+
+        while len(errors):
+            old_error = errors.popitem(last=False)
+            question_id, error_message = old_error
+
+            if section.get_question(question_id).type == 'boolean_list':
+                # errors don't get passed into a boolean list (or, therefore, its nested questions)
+                # unless a 'truthy' value exists for its parent id
+                new_errors[question_id] = True
+
+                for index, essential_question in enumerate(brief[question_id]):
+                    try:
+                        # check the response data for True/False values returned from boolean_list questions
+                        value_we_want_to_be_true_or_false = response_data[question_id][index]
+                    except (IndexError, KeyError):
+                        value_we_want_to_be_true_or_false = None
+
+                    if not isinstance(value_we_want_to_be_true_or_false, bool):
+                        # Each non-boolean value is an error
+                        boolean_question_id = "{}-{}".format(question_id, index)
+                        new_errors[boolean_question_id] = {
+                            'input_name': boolean_question_id,
+                            'message': error_message['message'],
+                            'question': essential_question
+                        }
```

Comment:
Gonna refactor the `ContentSummaryQuestion` to get some of this logic in there. 
It's not going to be the ideal solution, but it'll work.